.github/workflows/nightly-windows.yml

```diff
@@ -50,16 +81,16 @@ jobs:
       - name: Install Boost
         run: |
           Invoke-WebRequest `
-              "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.gz" `
-              -OutFile "boost_1_76_0.tar.gz" `
+              "${{ matrix.vc_boost.boost_url }}" `
+              -OutFile "${{ matrix.vc_boost.boost_archive_name }}" `
               -UserAgent "''"
-          tar xzf boost_1_76_0.tar.gz
-          rm boost_1_76_0.tar.gz
-          cd boost_1_76_0
+          tar xzf ${{ matrix.vc_boost.boost_archive_name }}
+          rm ${{ matrix.vc_boost.boost_archive_name }}
+          cd ${{ matrix.vc_boost.boost_folder_name }}
           .\bootstrap.bat
-          .\b2 address-model=${{ matrix.arch.address_model }} --with-thread --with-chrono install
+          .\b2 address-model=${{ matrix.arch.address_model }} --with-atomic --with-thread --with-chrono install
```

Comment:
what was the reason for atomic?

Reply:
We have dependency to `Boost.Thread` which transitively depends on `Boost.Atomic`. At olders versions, Boost doesn't complain about it but with newer versions it complains so that's why I added this flag.